drivers/usb/device/Kconfig

```diff
@@ -21,6 +21,31 @@ config USB_DW_IRQ_PRI
 	help
 	USB interrupt priority.
 
+config USB_DC_STM
```

Comment:
I would rename all *_STM or *_stm_* to *_STM32 and *_stm32_* to match the other STM32 drivers, but this is my personal preference not a must.

Reply:
I agree